Backpack/Dialog/Classes/BPKDialogIconView.m

```diff
@@ -49,19 +49,16 @@ - (CGSize)intrinsicContentSize {
     return [[self class] viewSize];
 }
 
-- (void)setIconBackgroundColor:(UIColor *_Nullable)iconBackgroundColor {
-    self.iconContainerView.backgroundColor = iconBackgroundColor;
+- (void)setIconDefinition:(BPKDialogIconDefinition *)iconDefinition {
+//    self.iconContainerView.backgroundColor = iconDefinition.iconBackgroundColor;
```

Comment:
For somereason it blows up on these lines, but I have no idea why 😢 
Feel like I've been looking at it too long to see the wood for the trees. Probably something simple that I'm missing @k0nserv @phelgo ?

Reply:
@k0nserv solved it for me 👍